### ISSUE TITLE
How to spawn workers over ssh?

### ISSUE THREAD

**Issue by KronosTheLate** [JuliaParallel/Dagger.jl#444]
Can I use Dagger to spawn processes on another computer connected over ssh? 

`Distributed.addprocs` has a method that "Add worker processes on remote machines via SSH.". Can spawn workers with this, schedule tasks on them with Dagger, and expect it to work?

A few sentences on ssh and computations on other computers in the docs would be lovely ^_^

**Comment by jpsamaroo** [comment]
Yes, you would use the standard `addprocs` from Distributed to do this. Feel free to add a sentence or two to the docs to point this out (maybe in the Quickstart section?) if you have time :slightly_smiling_face:
